mkdocs.yaml

```diff
@@ -25,6 +25,7 @@ nav:
     - Mondo sources:
       - Overview: sources.md
       - DO: sources/doid.md
+      - GARD: sources/gard.md
```

Comment:
@matentzn In `mkdocs.yaml`, I added an entry for GARD in the "Mondo sources" section. However, there is also a "Mondo source metrics" section, and I'm unsure what needs to / should be done to add GARD or metrics for any other new sources htere.

Reply:
seems you figured it put